bin/main.rb

```diff
@@ -0,0 +1,85 @@
+require '../lib/board.rb'
+$winner = nil
+$current_player = "Player x"
+$win_combination = [
+  #vertical
+ [0,1,2],[3,4,5],[6,7,8],
+ #horizontal
+ [0,3,6],[1,4,7],[2,5,8],
+ #diagonal
+ [0,4,8],[2,4,6]
+]
+
+private
+def turn
+  puts "#{$current_player}" 
+  position = gets.chomp.to_i - 1 
+  if $current_player == "Player x" 
+    @board[position] = "x"
+  else
+    @board[position] = "o" 
+  end
+end
+

```

Comment:
Can be a method of `Game` class. You could rewrite it like this

```ruby
def turn(position)
  if (@current_player == "Player x") # @current_player == 1
    @board[position] = "x"
  else
    @board[position] = "o"
  end
end
```

Reply:
done